ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -38,16 +38,16 @@
                     </span>
                 </span>
 
-                <!--Search Field-->
-                <span ng-controller="SearchCtrl" class="form-group form-inline" id="platform-job-text-search-field-parent">
-                    <input id="platform-job-text-search-field"
-                           title="Click to enter filter values"
+                <!--Quick Filter Field-->
+                <span ng-controller="SearchCtrl" class="form-group form-inline" id="quick-filter-parent">
+                    <input id="quick-filter"
+                           title="Click to enter quick filter values"
```

Comment:
This is customer facing so we don't need to change this, it can just say what it did prior. Our 'quick' filter changes are generally internal things.